apps/total-typescript/src/hooks/use-mux-player.tsx

```diff
@@ -59,15 +73,12 @@ export const VideoProvider: React.FC<
   onEnded = async () => {},
   section,
   exerciseSlug,
+  videoThumbId,
```

Comment:
this is the mux playback id as well and we just pass it around instead of rooting into the "exercise" for it

Reply:
i think i'm going to switch to the videoResourceId and create an api proxy endpoint for the video thumbnail and then this "mux" hook will be something like useVideoResource 🤔